app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -50,14 +50,13 @@ def initialize_cloud_inventory_collections
   # model_class defined due to ovirt dependency
   def add_vms
     add_collection_with_ems_param(cloud, :vms) do |builder|
-      builder.add_properties(:model_class => ManageIQ::Providers::Openstack::CloudManager::Vm)
```

Comment:
why is there nothing to replace this deletion like `builder.add_properties(:model_class => manager.class::Vm)` ?

Reply:
Because as long as the model_class matches the `manager_class::model_name` paradigm the model_class is automatically built.  You only have to do this if the model class doesn't match (e.g. the BaseTemplate)